internal/elements.go

```diff
@@ -342,7 +342,8 @@ type ETag string
 func (etag *ETag) UnmarshalText(b []byte) error {
 	s, err := strconv.Unquote(string(b))
 	if err != nil {
-		return fmt.Errorf("webdav: failed to unquote ETag: %v", err)
+		*etag = ETag(b)
```

Comment:
Hm, this completely ignores all errors. Not sure this is a great idea.

Reply:
Agreed. But how we can logged error with warn status?